apps/mochi-web/components/base/token-table-list/token-table-list.tsx

```diff
@@ -0,0 +1,85 @@
+import { utils as mochiUtils } from '@consolelabs/mochi-ui'
+import { ColumnProps, Table, TableProps, Typography } from '@mochi-ui/core'
+import { utils } from 'ethers'
+import { ModelBalance } from '~types/mochi-pay-schema'
+import { TokenAvatar } from '~cpn/base/token-avatar'
+import { Bag, WalletSolid } from '@mochi-ui/icons'
+
+interface Props extends Omit<TableProps<ModelBalance>, 'columns'> {}
+
+const Token: ColumnProps<ModelBalance>['cell'] = (props) => (
+  <div className="flex items-center space-x-2">
+    <TokenAvatar
+      src={props.row.original.token?.icon || ''}
+      name={props.row.original.token?.symbol || ''}
+      chainSrc={props.row.original.token?.chain?.icon || ''}
+      chainName={props.row.original.token?.chain?.symbol || ''}
+    />
+    <div>
+      <div className="flex space-x-1">
+        <Typography level="h8">
+          {mochiUtils.formatTokenDigit(
+            utils.formatUnits(
+              props.row.original.amount || 0,
+              props.row.original.token?.decimal,
+            ),
+          )}
+        </Typography>
+        <Typography level="h8" color="textSecondary">
+          {props.row.original.token?.symbol}
+        </Typography>
+      </div>
+      <Typography
+        level="p6"
```

Comment:
The two typo elements have different colors in the design. Can you double-check?

Reply:
Asking the designer. I think we can move on with the PR. If this needs to change, we will follow up with a subsequent update.

<img width="1026" alt="image" src="https://github.com/consolelabs/mochi-ui/assets/12707960/974e20a4-0a93-40a8-a879-13b69c4c0387">